src/VisualStudio/Core/Impl/LanguageServiceBrokerShim/LanguageServiceBrokerShim.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Threading;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageServiceBrokerShim
+{
+    [Export(typeof(ILanguageServiceBrokerShim))]
+    internal class LanguageServiceBrokerShim : ILanguageServiceBrokerShim
```

Comment:
what'st he value of the shim type versus just having the consumer of this import ILanguageServiceBroker2  instead?

Reply:
This shim type was created since we had to move the `Microsoft.VisualStudio.LanguageServer.Client.Implementation` package reference elsewhere. Context: https://github.com/dotnet/roslyn/pull/61784/files/847ba3384fea55421eec178837eba2fe0f5e65bf#r904221432